src/manifold/src/impl.h

```diff
@@ -108,7 +108,7 @@ struct Manifold::Impl {
   SparseIndices VertexCollisionsZ(VecView<const glm::vec3> vertsIn,
                                   bool inverted = false) const;
 
-  bool IsEmpty() const { return NumVert() == 0; }
+  bool IsEmpty() const { return NumTri() == 0; }
```

Comment:
Just curious, does this change affect anything?

Reply:
I was getting manifolds from `Decompose` that had one vertex and no triangles, but they weren't marked empty. I fixed the underlying problem too, but this still feels like a better definition.